src/pyj/read_book/flow_mode.pyj

```diff
@@ -3,10 +3,10 @@
 from __python__ import bound_methods, hash_literals
 
 from dom import set_css
-from read_book.globals import current_spine_item, get_boss
+from read_book.globals import current_spine_item, get_boss, rtl_page_progression, ltr_page_progression
```

Comment:
Two functions. LTR is usually used in cases where we want the default behavior to come first in a clause, RTL is used usually in cases where we override the value afterward.